{"uuid": "ce5a7663-ce81-4973-b467-7ec88f6931e8", "children": ["fbce1f18-5fcb-41a5-bb4e-adeb1a3e282c"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: Unable to find element with xpath == //a[.=\"Create Account\"]\n\n", "trace": "  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 949, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 775, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\patelvi\\PycharmProjects\\NCE\\onx\\test_multi.py\", line 64, in browser\n    driver.find_element_by_xpath('//a[.=\"Create Account\"]').click()\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 394, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 978, in find_element\n    'value': value})['value']\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1574168581527, "stop": 1574168657922}], "start": 1574168581527, "stop": 1574168659108}